bip-update-process.md

```diff
@@ -0,0 +1,648 @@
+    is inclusive of any BIPs that can be implemented and affect interoperability of Bitcoin applications. Since all BIPs
+    are individual recommendations by the authors (even if some may eventually achieve endorsement by the majority of
+    the community), the prior reminder that Informational BIPs are optional is dropped.
+[^comments]: **Why were comments, Comments-URI, and Comment-Summary removed from the process?**  
+    The comments feature saw insignificant adoption. Few BIPs received any comments and barely any more than two with
+    only a handful of contributors commenting at all. This led to many situations in which one or two comments ended up
+    dominating the comment summary. While some of those comments may have been representative of broadly held opinions, it
+    also overstated the importance of individual comments directly in the Preamble of BIPs. As collecting feedback in
+    this accessible fashion failed, the new process puts the onus back on the audience to make their own evaluation.
+[^layer]: **Why is the layer header now permitted for other BIP types?**  
+    The layer header had already been used by many Informational BIPs, so the rule that it is only available to
+    Standards Track BIPs is dropped.
+[^OtherImplementations]: **What is the issue with "Other Implementations" sections in BIPs?**  
+    In the past, some BIPs had "Other Implementations" sections that caused frequent change requests to existing BIPs.
+    This put an onus on the BIP authors, and frequently led to lingering pull requests due to the corresponding BIPs’
+    authors no longer participating in the process. Many of these alternative implementations eventually became
+    unmaintained or were low-quality to begin with. Therefore, "Other Implementations" sections are heavily discouraged.
+[^preliminary]: **Why has the Draft status been renamed to Preliminary?**  
+    The term "draft" appears all over this document and was inconsistently overlapping with the status field value.
+    Also, "Draft" stood out as the only noun among the status field values. Preliminary fits the intended meaning and
+    does not appear otherwise in the process description.
+[^ready]: **Why has the Proposed status been renamed to Ready?**  
+    Some reviewers of this BIP raised that in a process that outlines the workflow of Bitcoin Improvement _Proposals_
+    using "Proposed" as a status field value was overloading the term: clearly _proposals_ are proposed at all stages of
+    the process.
+[^rejection]: **Why can proposals remain in Preliminary or Ready indefinitely?**  
+    The automatic 3-year timeout of BIPs has led to some disagreement in the past and seems unnecessary in cases where
+    the author is still active in the community and still considers their idea worth pursuing. On the other hand,
+    proposals that appear stale may be tested and cleared out after only one year which hopefully achieves the main
```

Comment:
```suggestion
    proposals that appear stale may be tested and cleared out after only one year which achieves the main
```
Nit, this is more confident, but if that's too strong, maybe "which should achieve". The word "hopefully" expresses an emotion, which arguably is out of place in a document like this (just IMO).

Reply:
Thanks, I have amended this to 

```diff
-proposals that appear stale may be tested and cleared out after only one year which hopefully achieves the main
-goals of the original rule.
+proposals that appear stale may be tested and cleared out after only one year which should achieve the main
+goal of the original rule.
```